packages/api/src/external/carequality/patient-conversion.ts

```diff
@@ -0,0 +1,43 @@
+import { PatientDiscoveryRequest, XCPDGateways } from "@metriport/ihe-gateway-sdk";
+import { PurposeOfUse } from "@metriport/carequality-sdk";
+import { Patient as FHIRPatient } from "@medplum/fhirtypes";
+import { uuidv7 } from "@metriport/core/util/uuid-v7";
+import dayjs from "dayjs";
+
+export function patientToIheGateway({
```

Comment:
why don't we call this what it is? ie `createPatientDiscoveryRequest` or something